delfin/api/v1/storages.py

```diff
@@ -219,3 +218,20 @@ def _set_synced_if_ok(context, storage_id, resource_count):
         storage['sync_status'] = resource_count * constants.ResourceSync.START
         storage['updated_at'] = current_time
         db.storage_update(context, storage['id'], storage)
+
+
+def _is_same_access_info(access_info_a, access_info_b):
+    host_port_a = _get_host_port(access_info_a)
+    host_port_b = _get_host_port(access_info_b)
+    return host_port_a == host_port_b
+
+
+def _get_host_port(access_info):
```

Comment:
The method can be optimized as folllowed. It will be easy to enhance, if other connecting clients increases in future.

def _get_host_port(access_info):
     access_info_conn = ['rest', 'ssh']
     host_port = dict()
    for conn in access_info_conn:
        host_port[conn + '_host'] = access_info[conn]['host']
        host_port[conn + '_port'] = access_info[conn]['port']